src/OrderingForm.js

```diff
@@ -0,0 +1,240 @@
+import styled, { css } from 'styled-components'
+
+function OrderingForm() {
+  return (
+    <Wrapper>
+      <Headline>Rette mich, wer kann!</Headline>
+      <Form>
```

Comment:
Why didn't you add a `submit` event? With not adding it, this component is not really functional as yet.